lang/java/avro/src/main/java/org/apache/avro/SchemaNormalization.java

```diff
@@ -17,17 +17,22 @@
  */
 package org.apache.avro;
 
+import org.apache.avro.util.internal.JacksonUtils;
```

Comment:
This class is currently Jackson-independent... I think the JSON is being written manually here to ensure that it's isolated from any specific JSON implementations, and the trend of the last few versions has been to reduce the dependency on Jackson in particular.

Is it possible to avoid introducing this class?